XmlFormatterOsIndependent/Commands/BaseDataCommand.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace XmlFormatterOsIndependent.Commands
+{
+    /// <summary>
+    /// This class is a base class for any data driven command
+    /// </summary>
+    public abstract class BaseDataCommand : IDataCommand
+    {
+        /// <inheritdoc/>
+        public event EventHandler CanExecuteChanged;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused event 'CanExecuteChanged' or invoke it.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5634024)